src/components/inputs/money-input/money-input.js

```diff
@@ -329,34 +372,55 @@ export default class MoneyInput extends React.Component {
     if (this.props.onBlur) this.props.onBlur(event);
   };
 
+  amountInputRef = React.createRef();
+
   render() {
+    const hasNoCurrencies = this.props.currencies.length === 0;
+    const hasWarning = this.props.hasCurrencyWarning;
+    const hasError = this.props.hasCurrencyError;
+    const currencySelectStyles = createCurrencySelectStyles({
+      hasWarning,
+      hasError,
+      hasNoCurrencies,
+    });
+    const options = this.props.currencies.map(currencyCode => ({
+      label: currencyCode,
+      value: currencyCode,
+    }));
+
+    const option = do {
+      const matchedOption = options.find(
+        o => o.value === this.props.value.currencyCode
+      );
+      if (matchedOption) matchedOption;
+      // ensure an option is found, even when the currencies don't include
+      // the money value's currencyCode
+      else if (this.props.value.currencyCode.trim() !== '')
+        ({
+          label: this.props.value.currencyCode,
+          value: this.props.value.currencyCode,
+        });
```

Comment:
For myself: this is whenever a currency code was set which is now removed from the projects possible currencies.